### PR TITLE
New version: Microsoft.Teams.Preview version 1.6.00.21970

### PR DESCRIPTION
**Pull request by vedantmgoyal9**
###### Microsoft Reviewers: codeflow:open?pullrequest=https://github.com/microsoft/winget-pkgs/pull/115731&drop=dogfoodAlpha

### CODE REVIEW

**Comment by stephengillie** [comment]
Automatic Validation ended with:
image

File: manifests/m/Microsoft/Teams/Preview/1.6.00.21970/Microsoft.Teams.Preview.installer.yaml
Comment (CoolPlayLin) [inline manifests/m/Microsoft/Teams/Preview/1.6.00.21970/Microsoft.Teams.Preview.installer.yaml:1]: Please delete all of not necessary InstallerSwitches

**Review by CoolPlayLin — CHANGES_REQUESTED** [review]

File: manifests/m/Microsoft/Teams/Preview/1.6.00.21970/Microsoft.Teams.Preview.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# Created with WinGet Automation using Komac v1.10.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: Microsoft.Teams.Preview
+PackageVersion: 1.6.00.21970
+InstallerType: msi
+Scope: user
+InstallerSwitches:
+  Silent: /s
+  SilentWithProgress: /s
```
Comment (stephengillie) [inline manifests/m/Microsoft/Teams/Preview/1.6.00.21970/Microsoft.Teams.Preview.installer.yaml:10]:
```suggestion
  Silent: /quiet
  SilentWithProgress: /passive

```

Comment (stephengillie) [inline manifests/m/Microsoft/Teams/Preview/1.6.00.21970/Microsoft.Teams.Preview.installer.yaml]:
```suggestion
  Silent: /silent
  SilentWithProgress: /passive
```

**Pull request closed without merging**